app/views/curation_concerns/base/_form_representative.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="row">
```

Comment:
There's no need to add these templates to Sufia if they're a direct copy out of CurationConcerns. It can inherit them.